.github/workflows/auto-response.yml

```
@@ -1,29 +1,41 @@
 name: Issue/PR response
+
 permissions:
   issues: write
   pull-requests: write
+
 on:
   issues:
-    types:
-      - opened
+    types: [reopened, opened]
```

Why does it run on reopened?
```suggestion
    types: opened
```

---

I merged adding to project task into this ci file. It has to be run when reopened too. I added condition to adding auto comment not to run when reopened.